### ISSUE TITLE
Binding not found for BindingIdentifier

### ISSUE THREAD

**Issue by xxoo** [babel/babili#549]
environment
```
nodejs@7.10
babel-core@6.24.1
babel-preset-babili@0.1.1
babel-preset-env@1.5.1
```

test.js
```javascript
var babel = require('babel-core');
var t = babel.transformFileSync('code.js', {
	presets: [["env",{"targets":{"browsers":"ie >= 7"}}],"babili"],
	babelrc: false
});
```

code.js
```javascript
console.log({
	a: function(){}
});
```

command line
```
node test.js
```

result
```
/Users/xx/Desktop/fusion/dev/common/kernel/node_modules/babel-core/lib/transformation/file/index.js:590
      throw err;
      ^

Error: code.js: Binding not found for BindingIdentifier. aPlease report this at https://github.com/babel/babili/issues/new
    at BindingIdentifier (/Users/xx/Desktop/fusion/dev/common/kernel/node_modules/babel-plugin-minify-mangle-names/lib/index.js:216:21)
    at newFn (/Users/xx/Desktop/fusion/dev/common/kernel/node_modules/babel-traverse/lib/visitors.js:318:17)
    at bfsTraverse (/Users/xx/Desktop/fusion/dev/common/kernel/node_modules/babel-plugin-minify-mangle-names/lib/bfs-traverse.js:34:43)
    at Mangler.collect (/Users/xx/Desktop/fusion/dev/common/kernel/node_modules/babel-plugin-minify-mangle-names/lib/index.js:260:9)
    at Mangler.run (/Users/xx/Desktop/fusion/dev/common/kernel/node_modules/babel-plugin-minify-mangle-names/lib/index.js:70:14)
    at PluginPass.exit (/Users/xx/Desktop/fusion/dev/common/kernel/node_modules/babel-plugin-minify-mangle-names/lib/index.js:580:19)
    at newFn (/Users/xx/Desktop/fusion/dev/common/kernel/node_modules/babel-traverse/lib/visitors.js:276:21)
    at NodePath._call (/Users/xx/Desktop/fusion/dev/common/kernel/node_modules/babel-traverse/lib/path/context.js:76:18)
    at NodePath.call (/Users/xx/Desktop/fusion/dev/common/kernel/node_modules/babel-traverse/lib/path/context.js:48:17)
    at NodePath.visit (/Users/xx/Desktop/fusion/dev/common/kernel/node_modules/babel-traverse/lib/path/context.js:117:8)
```

**Comment by boopathi** [comment]
Thanks for the repro.

Duplicate #547

**Comment by boopathi** [comment]
Released in [0.1.2](https://github.com/babel/babili/releases/tag/babili%400.1.2)
